packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -447,15 +447,29 @@ export class WorkspaceRootUriAwareCommandHandler extends UriAwareCommandHandler<
         super(selectionService, handler);
     }
 
+    isVisible(): boolean {
+        return super.isVisible();
+    }
+
+    isEnabled(): boolean {
+       return super.isEnabled();
+    }
+
     protected getUri(): URI | undefined {
```

Comment:
it is so strange that we don't pass varargs here to a super implementation. Is not it a bug?